internal/pkg/otel/samples/darwin/logs_metrics_traces.yml

```diff
@@ -89,14 +322,52 @@ processors:
       - key: process.executable.path
         action: delete
 
-exporters:
-  otlp/apm:
-    endpoint: "${env:APM_ENDPOINT}"
-    headers:
-    # Elastic APM Server secret token or API key
-      Authorization: "Bearer ${env:APM_SECRET_KEY}"
+  elastictrace:
+  lsminterval:
+    intervals:
+      - duration: 1m
+        statements:
+          - set(resource.attributes["metricset.interval"], "1m")
+          - set(attributes["data_stream.dataset"], Concat([attributes["metricset.name"], "1m"], "."))
+          - set(attributes["processor.event"], "metric")
+      - duration: 10m
+        statements:
+          - set(resource.attributes["metricset.interval"], "10m")
+          - set(attributes["data_stream.dataset"], Concat([attributes["metricset.name"], "10m"], "."))
+          - set(attributes["processor.event"], "metric")
+      - duration: 60m
+        statements:
+          - set(resource.attributes["metricset.interval"], "60m")
+          - set(attributes["data_stream.dataset"], Concat([attributes["metricset.name"], "60m"], "."))
+          - set(attributes["processor.event"], "metric")
+  transform:
+    trace_statements:
+      #appending '-otel-collector' to all service name
+      - context: resource
+        statements:
+          - set(attributes["service.name"], Concat([attributes["service.name"], "otel-collector"], "-"))
+          - set(attributes["deployment.environment"], "otel-collector")
```

Comment:
@gregkalapos Do you know if we need to add the service name here? I thought it was for testing only so confirming if it is also needed for the actual config.

Reply:
Confirmed, it is not needed.